packages/xchain-polkadot/src/client.ts

```diff
@@ -35,6 +35,7 @@ class Client implements PolkadotClient, XChainClient {
   private phrase = ''
   private address: Address = ''
   private api: ApiPromise | null = null
+  private derivationPath = "44//354//0//0//0'"
```

Comment:
Does `//` (double slash) in path mean `hard-derived` [as described in documentation](https://substrate.dev/docs/en/knowledgebase/integrate/subkey#putting-it-all-together) of Polkadot's Subkey Tool?

Reply:
Yes, right